app/styles/ooyala/_article-featured-video.scss

```diff
@@ -0,0 +1,155 @@
+.article-featured-video {
+	.video-container {
+		background-size: cover;
+		padding-top: 56.25%;
+		position: relative;
+
+		&.player-ready {
+			.video-play-circle {
+				opacity: 1;
+			}
+
+			.video-details {
+				display: block;
+			}
+		}
+
+		&.is-played {
+			.video-details {
+				display: none;
+			}
+
+			.ooyala-article-video {
+				visibility: visible;
+			}
+		}
+	}
+
+	.wds-spinner {
+		$spinner-size: 59px;
+
+		left: 50%;
+		margin-left: -($spinner-size/2);
+		margin-top: -($spinner-size/2);
+		opacity: .7;
+		position: absolute;
+		top: 50%;
+
+		.wds-spinner__stroke {
+			stroke: $wds-color-white;
+		}
+	}
+
+	.ooyala-article-video {
+		height: 100%;
+		left: 0;
+		position: absolute;
+		top: 0;
+		width: 100%;
+	}
+
+	.video-placeholder {
+		-webkit-font-smoothing: antialiased;
+		bottom: 0;
+		color: $wds-color-white;
+		font-weight: bold;
+		left: 0;
+		position: absolute;
+		resize: none;
+		right: 0;
+		top: 0;
+	}
+
+	.video-play-circle {
+		align-items: center;
+		background-color: $wds-color-light-blue;
+		border: 6px solid rgba(0, 0, 0, 0.32);
+		border-radius: 50%;
+		bottom: 18px;
+		display: flex;
+		height: 80px;
+		justify-content: center;
+		opacity: .7;
+		position: absolute;
+		right: 18px;
+		width: 80px;
+
+		.wds-icon {
+			margin-left: 4px;
+			height: 30px;
```

Comment:
Can we talk with Design Team if we could use sizes defined in DS? Or maybe we should update DS?